apps/moose-console/src/app/infrastructure/databases/[databaseName]/tables/[tableId]/page.tsx

```diff
@@ -48,81 +51,60 @@ async function describeTable(databaseName: string, tableName: string): Promise<a
 async function getTable(databaseName: string, tableName: string): Promise<any> {
```

Comment:
Probably not worth doing here but this should be done in a follow up PR.

return type not being any and either a response or an error